user-access-handlers/src/main/java/no/unit/nva/handlers/UpdatePersonTermsConditionsHandler.java

```diff
@@ -0,0 +1,51 @@
+package no.unit.nva.handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.database.TermsAndConditionsService;
+import no.unit.nva.useraccessservice.model.TermsConditionsResponse;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.JacocoGenerated;
+
+import java.net.HttpURLConnection;
+
+public class UpdatePersonTermsConditionsHandler extends
+        ApiGatewayHandler<TermsConditionsResponse, TermsConditionsResponse> {
+
+    private final TermsAndConditionsService service;
+
+    @JacocoGenerated
+    public UpdatePersonTermsConditionsHandler() {
+        this(new TermsAndConditionsService());
+    }
+
+    public UpdatePersonTermsConditionsHandler(TermsAndConditionsService service) {
+        super(TermsConditionsResponse.class);
+        this.service = service;
+    }
+
+
+    @Override
+    protected void validateRequest(
+            TermsConditionsResponse input, RequestInfo requestInfo, Context context) throws ApiGatewayException {
+        requestInfo.getPersonCristinId();
+        requestInfo.getCurrentCustomer();
+    }
+
+    @Override
+    protected TermsConditionsResponse processInput(
+            TermsConditionsResponse input, RequestInfo requestInfo, Context context) throws ApiGatewayException {
+        return service.updateTermsAndConditions(
+                requestInfo.getPersonCristinId(),
+                input.termsConditionsUri(),
+                requestInfo.getCurrentCustomer()
+        );
+    }
```

Comment:
Nice, but since you have implemented a service for it and a new handler, we may not need a new table. Cause users-and-roles-table has already secondary index which lists all users by cristin id:
`List<UserDto> usersByCristinId = identityService.getUsersByCristinId(cristinId)`

Maybe use existing table for it? and persist it there? 
You still can use your service, but with another table. 

Simply agree terms for all users with provided cristinId?